### PR TITLE
[CloudPC] Rename hibernated to standbyMode in ShiftWorkCloudPcAccessState.

### PR DESCRIPTION
**Pull request by xinliu22**
roadmap item: 16826

### CODE REVIEW

**Review by RuiHou105 — APPROVED** [review]

**Review by AshleyYangSZ — APPROVED** [review]

**Comment by Danielabom** [comment]
Contacted author to get info.

**Comment by ningjingbogithub** [comment]
@Danielabom 
Roadmap: https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/16826/
Purpose/objective of this PR: We renamed "hibernated" to "standbyMode" because "hibernated" would be used in frontline CloudPC as another access state definition in the future. "standbyMode" can be more accurately express the state: "Cloud PC is in standby mode and can be used by the user at anytime without having to start the virtual machine beforehand."

File: api-reference/beta/resources/cloudpc.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ The following table lists the members of an [evolvable enumeration](#shiftworkcl
 |activating|Indicates that a user requested to connect the Cloud PC and the service is starting.|
 |waitlisted (deprecated)|Indicates that the shift work Cloud PC is in waitlisted state after the user requests to connect this Cloud PC and all shared use licenses are being actively used. This value is deprecated and will stop returning on May 17, 2023. |
 |unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|
-|hibernated|Indicates that the shift work Cloud PC is in the hibernated state before it's shut down and deallocated. A shift work Cloud PC in hibernated state is still accessible by the user.|
+|standbyMode|Indicates that the shift work Cloud PC is in the standby state before it's shut down and deallocated. A shift work Cloud PC in standby state is still accessible by the user.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:119]:
```suggestion
|standbyMode|Indicates that the shift work Cloud PC is in a standby state before it's shut down and deallocated. A shift work Cloud PC in standby state is still accessible by the user.|
```

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -3170,6 +3170,24 @@
...
-    }
+    },
+    {
+        "ChangeList": [
+          {
+            "Id": "2107ee19-7ed3-4480-936c-61a372e36751",
+            "ApiChange": "EnumType",
+            "ChangedApiName": "shiftWorkCloudPcAccessState",
+            "ChangeType": "Change",
+            "Description": "Rename the `hibernated` member to `standbyMode` in **shiftWorkCloudPcAccessState** enumeration type.",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:3181]:
```suggestion
            "Description": "Renamed the `hibernated` member to `standbyMode` in **shiftWorkCloudPcAccessState** enumeration type.",
```

**Review by Danielabom — APPROVED** [review]
@ningjingbogithub, thank you for working on this PR. PR Approved.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 3]
```diff
@@ -3170,6 +3170,24 @@
...
-    }
+    },
+    {
+        "ChangeList": [
+          {
+            "Id": "2107ee19-7ed3-4480-936c-61a372e36751",
+            "ApiChange": "EnumType",
+            "ChangedApiName": "shiftWorkCloudPcAccessState",
+            "ChangeType": "Change",
+            "Description": "Renamed the `hibernated` member to `standbyMode` in **shiftWorkCloudPcAccessState** enumeration type.",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:3181]:
```suggestion
            "Description": "Renamed the `hibernated` member to `standbyMode` in the **shiftWorkCloudPcAccessState** enumeration type.",
```

**Pull request merged**